Tools/eslint-config-cesium/browser.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+    'extends': 'cesium',
```

Comment:
none of the properties here need quotes, just the values.  Please do a once over of this PR and make that change everywhere.